server/src/test/java/org/spine3/server/storage/EventStorageShould.java

```diff
@@ -23,14 +23,24 @@
 import com.google.common.collect.ImmutableList;
```

Comment:
Can we please test another branch in the [EventStorage](https://codecov.io/gh/SpineEventEngine/core-java/src/event-query-field-filters/server/src/main/java/org/spine3/server/storage/EventStorage.java) (see line 313)?